platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLafPrefs.java

```diff
@@ -37,6 +38,8 @@ class FlatLafPrefs {
 
     private static final Preferences prefs = NbPreferences.forModule(FlatLafPrefs.class);
 
+    private static final boolean DEF_USE_WINDOW_DECORATIONS = SystemInfo.isWindows_10_orLater;
```

Comment:
@DevCharly Is it more correct to call SystemInfo.isWindows_10_orLater here, or FlatLaf.supportsNativeWindowDecorations()? E.g. in case FlatLAF later decided to make FlatLaf.supportsNativeWindowDecorations() true for Linux. (I suppose we could just change the constant in that case.)

Reply:
I guess I can answer this. There is no "native" window decoration for Linux. Linux has several windows managers. Each one with their own theme engine. Thus that will never happen.